crawlerr/__init__.py

```
@@ -13,8 +13,16 @@
 
-# Main exports
-from .server import mcp, main
+# Main exports - use lazy imports to avoid module conflicts
 from .config import settings
 
-__all__ = ["mcp", "main", "settings"]
\ No newline at end of file
+__all__ = ["settings"]
+
+# Lazy imports to prevent conflicts when running server module directly
+def get_mcp():
```

Ruff (0.12.2)

Missing return type annotation for private function `get_mcp`

(ANN202)